en/08_Changelogs/5.2.0.md

```diff
@@ -35,6 +36,33 @@ We have provided a severity rating of the vulnerabilities below based on the CVS
 - [CVE-2023-48714 - Record titles for restricted records can be viewed if exposed by GridFieldAddExistingAutocompleter](https://www.silverstripe.org/download/security-releases/cve-2023-48714) Severity: Medium
 - [CVE-2023-44401 - View permissions are bypassed for paginated lists of ORM data in GraphQL queries](https://www.silverstripe.org/download/security-releases/cve-2023-44401) Severity: Medium
 
+## Official support for `silverstripe/linkfield` {#linkfield}
+
+[`silverstripe/linkfield`](https://github.com/silverstripe/silverstripe-linkfield) 4.0.0 has been released and is now a commercially supported module.
+
+That means it is now covered by our [minor release policy](/project_governance/minor_release_policy/), our [major release policy](/project_governance/major_release_policy/), and our [security release process](/contributing/managing_security_issues).
+
+This module provides a [`Link`](api:SilverStripe\LinkField\Models\Link) model and two new form fields: [`LinkField`](api:SilverStripe\LinkField\Form\LinkField) and [`MultiLinkField`](api:SilverStripe\LinkField\Form\MultiLinkField). The intention of the module is to provide a clear and consistent approach to managing links in the CMS.
+
+This release of the module includes a bunch of features that were sorely missing from earlier versions including:
+
+- support for `has_many` relations
+- versioned support
+- validation
+- the ability to declare which link types are allowed per field
+- improved UI including better keyboard navigation.
```

Comment:
```suggestion
- Support for `has_many` relations
- Versioned support
- Validation
- The ability to declare which link types are allowed per field
- Improved UI including better keyboard navigation.
```

Reply:
The old company style guide was very explicit about list styles but since the new one is less strict I'll make this change.